src/PayU/Alu/Client.php

```diff
@@ -3,7 +3,9 @@
 namespace PayU\Alu;
 
 use PayU\Alu\Exceptions\ClientException;
-use PayU\Alu\Exceptions\ConnectionException;
+use PayU\PaymentsApi\Exceptions\AuthorizationException;
+use PayU\PaymentsApi\Exceptions\AuthorizationFactoryException;
+use PayU\PaymentsApi\Factories\AuthorizationFactory;
 use SimpleXMLElement;
```

Comment:
SimpleXMLElement import is not used anymore

Reply:
Removed it!